src/mocks/setupServer.js

```diff
@@ -0,0 +1,17 @@
+import { server } from './server';
+
+export const setupServer = function () {
+  beforeAll(() => {
+    server.listen({
+      onUnhandledRequest: 'warn'
```

Comment:
Why is that needed? I would advise treating every warning as an error in the tests as this may uncover potential problems with the production code (or the mocks).